src/Microsoft.AspNetCore.WebUtilities/WebEncoders.cs

```diff
@@ -156,7 +156,7 @@ public static byte[] Base64UrlDecode(string input, int offset, char[] buffer, in
 
         /// <summary>
         /// Gets the minimum <c>char[]</c> size required for decoding of <paramref name="count"/> characters
-        /// with the <see cref="Base64UrlDecode(char[], int, int)"/> method.
+        /// with the <see cref="M:Base64UrlDecode"/> method.
```

Comment:
Should be `Base64UrlDecode(string, int, char[], int, int)`. Definitely should not refer to the method group because the other overloads do not have a `char[]` parameter. (I neglected to update the signature here when I changed that method's parameters.)